gdp/small_batch/gdp_small_batch.py

```diff
@@ -128,45 +327,84 @@ def Y_exists_or_not(m, k, j):
 
     # Objective
     def obj_rule(m):
-        return sum(m.alpha[j]*(pe.exp(m.n[j] + m.beta[j]*m.v[j])) for j in m.j)
+        """
+        Objective: mininimize the investment cost [$].
+        Equation:
+            min z = sum(alpha[j] * exp(n[j] + beta[j]*v[j])) for j = mixer, reactor, centrifuge
+
+        Args:
+            m (pyomo.ConcreteModel): small batch GDP model
+
+        Returns:
+            Objective function (pyomo.Objective): Objective function to minimize the investment cost [$].
+        """
+        return sum(m.alpha[j] * (pe.exp(m.n[j] + m.beta[j] * m.v[j])) for j in m.j)
 
     m.obj = pe.Objective(rule=obj_rule, sense=pe.minimize)
 
     return m
 
 
 def external_ref(m, x, logic_expr=None):
+    """
+    Add the external variables to the GDP optimization problem.
+
+    Args:
+        m (pyomo.ConcreteModel): GDP optimization model
+        x (list): External variables
+        logic_expr (list, optional): Logic expressions to be used in the disjunctive constraints
+
+    Returns:
+        m (pyomo.ConcreteModel): GDP optimization model with the external variables
+    """
     ext_var = {}
     p = 0
     for j in m.j:
         ext_var[j] = x[p]
-        p = p+1
+        p = p + 1
 
     for k in m.k:
         for j in m.j:
             if k == ext_var[j]:
                 m.Y[k, j].fix(True)
-                m.Y_exists[k, j].indicator_var.fix(
-                    True)  # IS THIS REQUIRED????
-                m.Y_not_exists[k, j].indicator_var.fix(
-                    False)  # IS THIS REQUIRED????
+                # m.Y_exists[k, j].indicator_var.fix(
```

Comment:
Why is this commented out? Was it not necessary? In case not, that would answer the question in the comments.